core/src/main/java/bisq/core/api/CoreTradesService.java

```diff
@@ -273,15 +289,33 @@ Trade getTrade(String tradeId) {
                 ));
     }
 
+    List<Trade> getOpenTrades() {
+        coreWalletsService.verifyWalletsAreAvailable();
+        coreWalletsService.verifyEncryptedWalletIsUnlocked();
+        return tradeManager.getTrades();
```

Comment:
Why not mapping to `(TradeModel) t` here, to be consistent with `getTradeHistory` method at this level?

Reply:
Resolved by commit 4c8cec54f3247947dc211d50156b2e2af6663340.